code/src/components/TaskList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { Task } from './Task'
+import { List } from './styled components/TasksStyling'
+
+const TaskList = () => {
+    const items = useSelector((store) => store.tasks.items)
+
+    return (
+        <List>
+            {items.map((task) => (
+                <Task key={task.id} task={task} completed={task.completed} />
+            ))}
+        </List>
+    )
+}
+
+export default TaskList
```

Comment:
Your tasklist is so clean! And i really like the idea that you have a separate component for the task list and for the task item it self! I will maybe steal this idea for my own project 😉